pkg/pipeline/extract/conntrack/aggregator.go

```diff
@@ -179,11 +185,13 @@ func (agg *aMax) update(conn connection, flowLog config.GenericMap, d direction,
 }
 
 func (cp *aFirst) update(conn connection, flowLog config.GenericMap, d direction, isNew bool) {
-	if isNew {
+	if isNew && flowLog[cp.inputField] != nil {
```

Comment:
I assume if we don't allow by default empty fields then we won't need this check as well as the one in the update function ?

Reply:
The goal here is to avoid reporting empty fields for `first` and `last` operations (just below this one). 
Without it, these will create field with nil: 
https://github.com/netobserv/flowlogs-pipeline/blob/381890dd4cf93fadebc491e7beb5c4165a9ceff7/pkg/pipeline/extract/conntrack/conn.go#L66

I'm not sure to get your point. Can you please clarify ?

Reply:
After testing it seems better to work directly in the `toGenericMap` function
https://github.com/netobserv/flowlogs-pipeline/pull/470/commits/74611d21c7b2a45686c4f754124f30c6fa05c19b

That avoid to manage per aggregate cases

Reply:
@msherif1234 any reason not skipping zeros for Bytes and Packets fields ? 
https://github.com/netobserv/flowlogs-pipeline/pull/470/commits/de53e15aa87a4cc7a398a91e9c7c6e0adce0a213

For consistency it would be useful

Reply:
no @jpinsonneau it makes sense to skip empty fields